pages/api/graphql.js

```diff
@@ -1,71 +0,0 @@
-// pages/api/graphql.js
-import { ApolloServer, gql } from 'apollo-server-micro';
-import { makeExecutableSchema } from 'graphql-tools';
-import { MongoClient } from 'mongodb';
-
-require('dotenv').config();
-
-const typeDefs = gql`
-  type User {
-    id: ID!
-    firstName: String!
-    lastName: String!
-    blog: String
-    stars: Int
-  }
-
-  type Query {
-    users: [User]!
-  }
-`;
-
-const resolvers = {
-  Query: {
-    users(_parent, _args, _context, _info) {
-      return _context.db
-        .collection('test')
-        .findOne()
-        .then((data) => {
-          return data.users;
-        });
-    }
-  }
-};
-
-const schema = makeExecutableSchema({
-  typeDefs,
-  resolvers,
-  introspection: true,
-  playground: true
```

Comment:
Can you also try adding these in and see if they allow us to use the playground? Should look something like this: 
![image](https://user-images.githubusercontent.com/2253624/102928590-d31d1080-444d-11eb-8f71-056442a0c3cb.png)

Reply:
Or maybe it already works? Try hitting localhost:3000/api/graphql to see if this pops up, I was trying in the deployed preview, I think it's disabled there. But this will be useful in the future when we're building more queries so we can double check our queries give back what we expect instead of guessing and regenerating a million times haha.

Reply:
![image](https://user-images.githubusercontent.com/47007049/102929415-660a7a80-444f-11eb-8d27-457ec863db78.png)
looks like it already works

Reply:
nice!